scripts/archive.txt

```diff
@@ -12,5 +12,7 @@ libraries/core/README.md
 libraries/util/README.md
 tests/binaryfile-perf-test/README.md
 tests/binaryfile-perf-test/nassau-binaryfile-perf-test.jar
-tests/soupbintcp-perf-test/README.md
-tests/soupbintcp-perf-test/nassau-soupbintcp-perf-test.jar
+examples/soupbintcp-client/README.md
+examples/soupbintcp-client/nassau-soupbintcp-client.jar
+examples/soupbintcp-server/README.md
+examples/soupbintcp-server/nassau-soupbintcp-server.jar
```

Comment:
A nit: let's keep these in the alphabetical order. 🤓

Reply:
Done